topics/materials-science/tutorials/muspinsim/tutorial.bib

```diff
@@ -0,0 +1,45 @@
+
+# This is the bibliography file for your tutorial.
+#
+# To add bibliography (bibtex) entries here, follow these steps:
+#  1) Find the DOI for the article you want to cite
+#  2) Go to https://doi2bib.org and fill in the DOI
+#  3) Copy the resulting bibtex entry into this file
+#
+# To cite the example below, in your tutorial.md file
+# use {% cite Batut2018 %}
+#
+# If you want to cite an online resourse (website etc)
+# you can use the 'online' format (see below)
+#
+
+@software{muspinsim,
+    author = {Sturniolo, Simone and Liborio, Leandro and Owen, Josh and Mudaraddi, Anish and {Muon Spectroscopy Computational Project}},
+    license = {MIT},
+    title = {{muspinsim}},
+    url = {https://github.com/muon-spectroscopy-computational-project/muspinsim},
+    version = {v1.1.0},
+    month = {5},
+    year = {2022},
+    doi = {10.5281/zenodo.6563074}
+}
+
+@article{nelder_mead_1965,
+    title={A Simplex Method for Function Minimization},
+    volume={7},
+    doi={10.1093/comjnl/7.4.308},
+    number={4},
+    journal={The Computer Journal},
+    author={Nelder, J. A. and Mead, R.},
+    year={1965},
+    pages={308-313}
+}
+
+@book{nocedal_wright_2006,
+    place={New York (NY)},
+    title={Numerical Optimization},
+    publisher={Springer},
+    author={Nocedal, Jorge and Wright, Stephen J},
+    year={2006},
+    url={https://www.csie.ntu.edu.tw/~r97002/temp/num_optimization.pdf}
```

Comment:
Not sure if this is something we can fix but the tilde in this URL is interpreted as a space (%C2%A0) on the site so it doesn't actually work. Would an escape character %7E work?

Reply:
That's my bad, I didn't bother checking it worked from the tutorial link. Percent encoding does seem to work, great suggestion.